### ISSUE TITLE
Mouse erratic behaviour on 1.8.4

### ISSUE THREAD

**Issue by bcdonadio** [symless/synergy#5660]
### Operating Systems ###

Client: Fedora 24 64-bit

Server: Fedora 24 64-bit

### Synergy Version ###

1.8.4

### Steps to reproduce bug ###

1. Install 1.8.4 on server and client on F24.
2. Start server and client.
3. Mouse erratic behaviour occurs

### Other info ###

* When did the problem start to occur? When I updated to 1.8.4. On 1.8.3 everything is fine.
* Is there a way to work around it? Yes, downgrade to 1.8.3.
* Does this bug prevent you from using Synergy entirely? Yes.

Here's a video of the issue happening on the client (the issue doesn't happen on the server screen): [![Synergy v1.8.4 mouse issue](https://img.youtube.com/vi/tJVorD3zD7k/0.jpg)](https://www.youtube.com/watch?v=tJVorD3zD7k)

**Comment by nlyan** [comment]
There were no changes made to mouse behaviour in 1.8.4 for Linux users, @XinyuHou any ideas?

**Comment by nlyan** [comment]
@bcdonadio Can you try the build that will shortly be available here:

http://symless.com/nightly?filter=issue5660

**Comment by bcdonadio** [comment]
@nylan Indeed, this build seems to have fixed the issue. I didn't test further the 1.8.4-0 build because I had work to do, and spare time was definetely scarce, therefore there could be other things not working as well.

With this f5c9bd8 build I'm having no problems so far. Thanks!

**Comment by TinLe** [comment]
I don't mean to hijack this thread, but I saw this behavior of the mouse jumping around also with 1.8.4 release on OSX 10.11.6 (client) to RHEL 6.6 (server).

I am now reverted back to 1.8.3 since that works most of the time.

**Comment by bcdonadio** [comment]
Altough I have absolutely zero evidence so far, I bet this issue is probably related to the support of dynamic monitor configuration. That part of X is *really* messy.
